### PR TITLE
Add missing commands for update management in sudoers

### PR DESCRIPTION
**Pull request by shuliums**
@robbiezhang @Atchub @vrdmr @Microsoft/omsagent-devs

### CODE REVIEW

File: installer/conf/sudoers [hunk 1]
```diff
@@ -32,10 +32,17 @@ omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsagent/ruby/bin/ruby /opt/microsof
...
-omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get update 
-omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get dist-upgrade * 
-omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get upgrade *  
+omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get dist-upgrade *
+omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get update
+omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get upgrade *
+omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get * update
+omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get * dist-upgrade
+omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get * upgrade
+omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get * install
```
Comment (Diastro) [inline installer/conf/sudoers]: What's the use of this cmd in the runbook? I thought we explicitly didn't want to whitelist install (I though we agreed on only whitelisting --only-upgrade install...

Comment (shuliums) [inline installer/conf/sudoers]: We have 3 commands perform install related actions. There is an ongoing discussion to determine which one can be removed.  I will update this pull request once we nail down the final list. 

apt-get -d -y install
apt-get -y --only-upgrade true install
apt-get -y --only-upgrade true -s install

**Comment by shuliums** [comment]
@Diastro Please reviews the changes.  removed omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get * install

**Comment by Diastro** [comment]
Thank you for addressing these!

**Pull request merged**